<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EqualizerDialog</class>
 <widget class="QDialog" name="EqualizerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>473</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>493</width>
    <height>473</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>493</width>
    <height>473</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Equalizer Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QRadioButton" name="eq3BandButton">
       <property name="maximumSize">
        <size>
         <width>140</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>3-Band Equalizer</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="eq10BandButton">
       <property name="text">
        <string>10-Band Equalizer</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="eqSaveButton">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Receive Equalizer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>180</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="text">
           <string>15dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>0dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string>-12dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="3">
       <widget class="QSlider" name="rxEq63Slider">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="11">
       <widget class="QSlider" name="rxEq16KSlider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="12">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="text">
           <string>15dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>0dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string>-12dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="rxEq32Slider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Preamp</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="rxEq32Label">
        <property name="text">
         <string>32</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QSlider" name="rxEq4KSlider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="10">
       <widget class="QSlider" name="rxEq8KSlider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QSlider" name="rxEq500Slider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSlider" name="rxEqPreampSlider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSlider" name="rxEq125Slider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QSlider" name="rxEq250Slider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QSlider" name="rxEq1KSlider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QSlider" name="rxEq2KSlider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="rxEq63Label">
        <property name="text">
         <string>63</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="rxEq125Label">
        <property name="text">
         <string>125</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="rxEq500Label">
        <property name="text">
         <string>500</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="rxEq250Label">
        <property name="text">
         <string>250</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="rxEq1kLabel">
        <property name="text">
         <string>1K</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="rxEq2kLabel">
        <property name="text">
         <string>2K</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="rxEq4kLabel">
        <property name="text">
         <string>4K</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QLabel" name="rxEq8kLabel">
        <property name="text">
         <string>8K</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLabel" name="rxEq16kLabel">
        <property name="text">
         <string>16K</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="rxEqResetButton">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>19</height>
      </size>
     </property>
     <property name="text">
      <string>Transmit Equalizer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>180</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="text">
           <string>15dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>0dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_22">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string>-12dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="3">
       <widget class="QSlider" name="txEq63Slider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="11">
       <widget class="QSlider" name="txEq16KSlider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="12">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="text">
           <string>15dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>0dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_25">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>29</height>
           </size>
          </property>
          <property name="text">
           <string>-12dB</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignHCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="txEq32Slider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QSlider" name="txEq4KSlider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="10">
       <widget class="QSlider" name="txEq8KSlider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QSlider" name="txEq500Slider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSlider" name="txEqPreampSlider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSlider" name="txEq125Slider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QSlider" name="txEq250Slider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QSlider" name="txEq1KSlider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="8">
       <widget class="QSlider" name="txEq2KSlider">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="txEq32Label">
        <property name="text">
         <string>32</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_27">
        <property name="font">
         <font>
          <pointsize>8</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Preamp</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="txEq63Label">
        <property name="text">
         <string>63</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="txEq125Label">
        <property name="text">
         <string>125</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="txEq250Label">
        <property name="text">
         <string>250</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="txEq500Label">
        <property name="text">
         <string>500</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="txEq1kLabel">
        <property name="text">
         <string>1K</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="txEq2kLabel">
        <property name="text">
         <string>2K</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="txEq4kLabel">
        <property name="text">
         <string>4K</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QLabel" name="txEq8kLabel">
        <property name="text">
         <string>8K</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLabel" name="txEq16kLabel">
        <property name="text">
         <string>16K</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="txEqResetButton">
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
